### PR TITLE
feat: profile page added with logout btn

### PR DESCRIPTION
**Pull request by tushartiwari7**
Added Profile page that shows
- User name
- Date when joined in
- Logout Btn

localhost_3001_profile

### CODE REVIEW

File: src/components/ProductCard/ProductCard.css [hunk 1]
```diff
@@ -15,18 +15,23 @@
...
+  backdrop-filter: none;
+}
+
 .card-description {
   height: 7rem;
 }
 
 .card:hover .card-content {
-  background: rgba(255,255,255,0.5);
+  background: rgba(255, 255, 255, 0.7);
```
Comment (sushil-mod) [inline src/components/ProductCard/ProductCard.css:34]: if possible use color variable

**Review by sushil-mod — APPROVED** [review]
nice code ,auth working fine

**Review by harshalini — APPROVED** [review]
Nice code, good to merge

File: src/components/ProductCard/ProductCard.css (hunk 1 above)
Comment (tushartiwari7) [reply src/components/ProductCard/ProductCard.css:34]: This css gives a glassy look to my content which is not possible with hex values.

**Pull request merged**